cmake/modules/FindSpiderMonkey.cmake

```diff
@@ -93,8 +93,8 @@ check_cxx_source_compiles(
 	"#include <jsapi.h>
 	int main()
 	{
-    JSRuntime *rt = JS_NewRuntime(8L * 1024L * 1024L);
-		if (rt != NULL)
+    JSContext *cx = JS_NewContext(JS::DefaultHeapMaxBytes);
```

Comment:
please explain this change

Reply:
This checks if that source compiles, but the original check was failing because it was based on a much older version of SpiderMonkey than the one we use now.